printer_mods/Shteeeeeve/V01-LED-Tophat/README.md

```diff
@@ -0,0 +1,40 @@
+# V0.1 LED Tophat
+
+![Image 1](Images/V01-LED-Tophat-01.jpg)
+
+## Printed Parts:
+1. LED Tophat - Main Body
+2. LED Tophat - Top Cover
+3. LED Tophat - Logo Insert
+
+## Parts:
+1. 4x M3x6 BHCS
+2. 4x M3x5x4 Heat Inserts
+3. 24V LED Lightstrip \\\ URL: https://tinyurl.com/7axzaff4
```

Comment:
Please replace this with a plain Amazon link, looks like it should be

    https://www.amazon.ca/gp/product/B074XCKJYB